messaging/src/main/java/org/axonframework/eventhandling/scheduling/quartz/FireEventJob.java

```diff
@@ -107,6 +107,7 @@ private EventMessage<?> createMessage(Object event) {
                 ? new GenericEventMessage<>(((EventMessage<?>) event).name(),
                                             ((EventMessage<?>) event).getPayload(),
                                             ((EventMessage<?>) event).getMetaData())
-                : new GenericEventMessage<>(QualifiedNameUtils.fromClassName(event.getClass()), event);
+                : new GenericEventMessage<>(QualifiedNameUtils.fromClassName(event.getClass()),
+                                            event); // TODO #3085 use MessageNameResolver
```

Comment:
Trusting this will be done in a follow-up PR, right?

Reply:
Yes, whenever I find some places where it's needed I left the comment :) 
There is a lot usages of the QualifiedNameUtils :O